adversaries.yml

```diff
@@ -21,6 +22,7 @@
           - a0676fe1-cd52-482e-8dde-349b73f9aa69
     - name: mission2 (stockpile)
       description: PowerShell discovery on a single machine
+      locked: 0
```

Comment:
should locked = 1 (for locked) and locked = 0 (for unlocked)? this seems to be a more natural fit for this flag.

Reply:
The scheme is 1 for locked and 0 for unlocked, just as you suggested. I changed these to 0 due to our "not really supporting this right now" approach to protection/locking, but I'll change them back to 1 so things loaded from file are protected.